_sass/ecosystem.scss

```diff
@@ -137,3 +137,160 @@
     border-color: #ecedf1;
   }
 }
+
+.ecosystem  {
+  .join-link {
+    color: inherit;
+    text-decoration: underline;
+  }
+
+  .join-notice {
+    text-align: center;
+    padding-top: rem(20px);
+    padding-bottom: rem(40px);
+
+    p {
+      color: $dark_grey;
+      margin-bottom: 0;
+      line-height: rem(30px);
+    }
+  }
+
+  .join-jumbotron {
+    @include desktop {
+      height: 262px;
+    }
+    border-bottom: 1px solid #C3C3C3;
+    width: 90%;
+
+    .container {
+      max-width: 920px;
+    }
+
+    h1 {
+      padding-top: rem(5px);
+      color: $black;
+      span {
+        font-weight: 300;
+        color: #812CE5;
+      }
+    }
+  }
+
+  .join-wrapper {
+    background-color: $white;
+
+    .main-content {
+      @include desktop {
+        padding-top: 4.4rem;
+      }
+    }
+
+    .container {
+      max-width: 920px;
+    }
+  }
+
+  .join-intro {
+    color: $dark_grey;
+    line-height: 28px;
+  }
+
+  .requirements {
+
+    span {
+      color: $black;
+      font-weight: bold;
+    }
+
+    .join-number {
+      color: $purple;
+      display: flex;
+      align-items: center;
+      @include desktop {
+        padding-left: rem(10px);
+      }
+    }
+
+    p {
+      margin-bottom: 0;
+      margin-top: rem(-7px);
+      @include desktop {
+        padding-left: rem(24px);
+      }
+    }
+
+    .col-md-11 {
+      @include desktop {
+        border-left: 2px solid $light_grey;
+      }
+    }
+  }
+
+  .row.requirements {
+    padding-bottom: rem(40px);
+  }
+
+  .ecosystem-form {
+    padding-bottom: rem(48px);
+
+    .field-group {
+      padding-bottom: rem(50px);
+    }
+
+    h3 {
+      text-transform: uppercase;
+      padding-top: rem(25px);
+      padding-bottom: rem(30px);
+    }
+
+    label {
+      color: $dark_grey;
+    }
+
+    input, textarea {
```

Comment:
When clicking on a form field in Chrome there is a blue outline- let's kill that